labs/perun-framework.md

```diff
@@ -1,14 +1,14 @@
 # Lab Name
 
-Direct State Transfer / Perun
+Perun Framework
 
 # Short Description
 
 Blockchain-agnostic State Channels framework, written in Go.
 
 # Scope of Lab
 
-Perun / Direct State Transfer is a Go implementation of the [Perun state
+Perun Framework is a Go implementation of the [Perun state
```

Comment:
```suggestion
The Perun Framework is a suite of implementations around the [Perun state
```
I would not mention implementation details like Go in the general description of the Perun framework. For example, The Ethereum smart contracts are written in Solidity, its tests are written in TypeScript etc. In the future, we might also add additional implementations in other languages.

Reply:
@sebastianst, thank you for the review! Your suggested change makes sense. I've added it in commit 3a1a883.